applications/GeoMechanicsApplication/custom_processes/set_absorbing_boundary_parameters_process.hpp

```diff
@@ -65,90 +60,29 @@ class SetAbsorbingBoundaryParametersProcess : public Process
         KRATOS_CATCH("")
     }
 
-    ///------------------------------------------------------------------------------------
-
-    /// Destructor
-    ~SetAbsorbingBoundaryParametersProcess() override {}
-
-///----------------------------------------------------------------------------------------------------------------------------------------------------------------------------------
-
-    /// Execute method is used to execute the ApplyConstantPhreaticLinePressureProcess algorithms.
-    void Execute() override
-    {
-    }
+    SetAbsorbingBoundaryParametersProcess(SetAbsorbingBoundaryParametersProcess const& rOther) = delete;
+    SetAbsorbingBoundaryParametersProcess& operator=(SetAbsorbingBoundaryParametersProcess const& rOther) = delete;
+    ~SetAbsorbingBoundaryParametersProcess() override = default;
 
     /// this function is designed for being called at the beginning of the computations
     /// right after reading the model and the groups
     void ExecuteInitialize() override
     {
         KRATOS_TRY
 
-        if (mrModelPart.NumberOfConditions() > 0) {
-
-                block_for_each(mrModelPart.Conditions(), [&](Condition& rCondition) {
-                    rCondition.SetValue(ABSORBING_FACTORS, mAbsorbingFactors);
-                    rCondition.SetValue(VIRTUAL_THICKNESS, mVirtualThickness);
-                });
-        }
+        block_for_each(mrModelPart.Conditions(), [&](Condition& rCondition) {
+            rCondition.SetValue(ABSORBING_FACTORS, mAbsorbingFactors);
+            rCondition.SetValue(VIRTUAL_THICKNESS, mVirtualThickness);
+        });
 
         KRATOS_CATCH("")
     }
 
-    /// Turn back information as a string.
-    std::string Info() const override
-    {
-        return "SetAbsorbingBoundaryParametersProcess";
-    }
```

Comment:
I believe this member function was accidentally deleted.

Reply:
Indeed, restored.